app/code/Transoft/Blog/etc/config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <catalog>
+            <blog>
+                <blog_applied_to>Type</blog_applied_to>
```

Comment:
Please specify the correct product types

Reply:
fixed